util/src/main/java/org/killbill/billing/util/export/dao/DatabaseExportDao.java

```diff
@@ -246,5 +253,18 @@ public Void withHandle(final Handle handle) throws Exception {
         });
     }
 
+    private boolean areAccountIdTenantIdColsPresent(List<ColumnInfo> columnsForTable, TableType tableType) {
+        boolean accountIdColPresent = false;
+        boolean tenantIdColPresent = false;
+        for(ColumnInfo column : columnsForTable) {
```

Comment:
We already did a loop through column name in line 152, could we leverage the same loop - or do we need to do it twice?

Reply:
Addressed in 05d1e14.